plugins/inputs/opcua/opcua_test.go

```diff
@@ -23,44 +25,70 @@ type OPCTags struct {
 	Want           interface{}
 }
 
-const servicePort = "4840"
+func MapOPCTag(tags OPCTags) (out input.NodeSettings) {
+	out.FieldName = tags.Name
+	out.Namespace = tags.Namespace
+	out.IdentifierType = tags.IdentifierType
+	out.Identifier = tags.Identifier
+	return out
+}
 
 func TestGetDataBadNodeContainerIntegration(t *testing.T) {
 	if testing.Short() {
 		t.Skip("Skipping integration test in short mode")
 	}
 
-	container := testutil.Container{
-		Image:        "open62541/open62541",
-		ExposedPorts: []string{servicePort},
-		WaitingFor:   wait.ForListeningPort(nat.Port(servicePort)),
+	// Spin-up the container
+	ctx := context.Background()
+	req := testcontainers.GenericContainerRequest{
```

Comment:
Out of curiosity, why did you switch from using the testutil container wrapper to using containers directly?

Reply:
That's a remainder from back in March when I branched from master. The change to testutil containers was made in May. I changed it to using the testutil.